cserver.py

```diff
@@ -217,7 +217,7 @@ def query():
     #check if owner has an active sale record or request
     sales = db.session.query(Sale).filter(Sale.owner == owner).count()
     res = []
-    if sales == 0:
+    if sales == 0 and string != 'ratings':
```

Comment:
You're right. For some reason I had to do that when testing initially, but now I do have a sales record in the database apparently.